resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -118,27 +109,24 @@ get_status_q()
 do_restart()
 {
     do_stop
-    case "$?" in
-        0|1)
```

Comment:
Sometimes, I think.  Today, if the `start-stop-daemon` fails with a retval of 2, 'end of schedule reached and processes were still running', the `restart` won't try to move on to the `start` and will return a `1` instead.  I suppose there could be other reasons that the stop might fail today where the start would still be attempted but I'm not coming up with any good ones.  With the new stop subcommand as it is on the PR, we wouldn't be able to distinguish the reasons a stop would fail (note that if the service had already been stopped, the result of the stop would be a success/0), so I just unconditionally have it not move on to the start.  Are you thinking something needs to be changed here?